src/adv.py

```diff
@@ -38,14 +39,47 @@
 #
 
 # Make a new player object that is currently in the 'outside' room.
-
+player = Player(input("What is your name, brave soul : "), room['outside'])
+# player = input("What is your name, brave soul : "), room['outside']
+# print(player)
 # Write a loop that:
-#
-# * Prints the current room name
+# player_cmd = ""
+# while player_cmd not in ():
+while True:
+    # * Prints the current room name
+    time.sleep(1)
```

Comment:
Comment on why you decided to use time.sleep

Reply:
adds suspense??? more experimenting